docs/security.md

```diff
@@ -0,0 +1,154 @@
+     1. The account can be locked either on Storefront or in VC admin by clicking the 'Lock account' button in the menu bar.
+     1. Possibility to unlock the locked account by clicking the 'Unlock account' button placed in the menu bar.
+1. Container store (not editable):
+     1. Container property is defined via API;
+     1. Defines the stores in which the account was created and to which the account has access.
+
+![User information](media/screen-user-information.png)
+
+#### Assign Role to Account
+
+1. Open user's account and click on the 'Roles' widget;
+1. On 'Manage roles' blade click 'Assign';
+1. Select the role to assign and click the 'OK' button
+1. Save the changes;
+1. The new role will appear in user's account
+
+### Generate API key
+
+The API key should be linked to the security account.
+
+**Important!**
+
+1. If the API key was de-activated, the system will not allow the API call using the old key;
+1. If the API key was de-activated and then re-activated, the system will not allow use the old key. A new API key will be generated.
+
+To generate an API key, you should go through the following steps:
+
+1. Open the security account;
+1. Select the API key widget;
+1. Click the 'Generate' button in the menu bar;
+1. Activate the key by switching on the 'Is Active' button.
+![API key](media/screen-api-key.png)
+
+#### Assign permissions
+
+1. Open the user's account and select 'Roles';
+1. On 'Manage roles' blade select the role to which the permissions should be assigned;
+1. On 'Roles details' blade click 'Assign';
+1. Select the permissions to assign to the selected role and save the changes.
+1. The selected permissions will be assigned to the role.
+![Assign permissions](media/screen-assign-permissions.png)
+
+#### Create New Role
+
+1. Go to More->Security->Roles and click 'Add';
+1. Enter the Role name;
+1. Enter the Role description if needed;
+1. Click 'Create';
+1. The new Role will be created and displayed on the 'Roles' blade.
+
+![Roles](media/screen-roles.png)
+
+![Create New Role](media/screen-create-new-role.png)
+
+#### Assign Permissions to Role
+
+1. Select the Role on 'Roles' blade and click 'Assign';
+1. Select permissions to assign to the role;
+1. Click 'OK';
+1. Save the changes;
+1. The selected permissions will be assigned to the role.
+
+![Assign permissions to role](media/screen-assign-permissions-to-role.png)
+
+#### Remove Assigned Permissions
+
+1. Select the Role on 'Roles' blade 
+1. Under 'Assigned Permissions' check the permissions you want to remove from the selected role;
+1. Click 'Remove' and save the changes;
+1. The permissions will be removed from the role.
+
+![Remove permissions](media/screen-remove-permissions.png)
+
+## Documentation
+
+https://github.com/VirtoCommerce/vc-platform/blob/release/3.0.0/src/VirtoCommerce.Platform.Web/appsettings.json#L56
+
+https://community.virtocommerce.com/t/make-secure-web-api-for-platform-3-0/105
```

Comment:
Change to 
[Make secure Web API for Platform 3.0](https://community.virtocommerce.com/t/make-secure-web-api-for-platform-3-0/105)